ci/scripts/check-no-bazelrc-site.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Checks that .bazelrc-site file has not been accidentally committed to git
+# where it could potentially impact the local configuration of other users.
+#
+# To be run in a CI environment (with a repo), and thus we simply check for
+# the non-existence of the file, to confirm that it is not in the repo.
+
+set -e
+
+if git .bazelrc-site ]; then
+    echo -n "##vso[task.logissue type=error]"
+    echo "The .bazelrc-site file should appear in a clean checkout"
```

Comment:
```suggestion
    echo "The .bazelrc-site file should not appear in a clean checkout."
```
?

Reply:
Thanks for the catch.

Sure I can add a comment about this in the docs.

Reply:
Done!